src/Extracting_data.ts

```diff
@@ -0,0 +1,58 @@
+import sql from './db.js'
+export class LocationDatabase {
+    async GetLocations(): Promise<{ imsi: number, x: number, y: number, calctime: number }> {
+        const Locations = await sql`
+        select
+            imsi,
+            x,
+            y,
+            calctime
+        from
+            Location`;
+
+        return Locations as any
+    }
+
+    async GetMeasurements(): Promise<{ mid: number, imsi: number, aid: number, timestamp: number, strengthDBM: number }> {
+        const Measurements = await sql`
+            select 
+                mid,
+                imsi,
+                aid,
+                timestamp,
+                StrengthDBM
```

Comment:
This is not the same as above
```suggestion
                strengthDBM
```